source/Planet.cpp

```diff
@@ -639,21 +678,36 @@ bool Planet::IsUnrestricted() const
 // but do so with a less convoluted syntax:
 bool Planet::HasFuelFor(const Ship &ship) const
 {
-	return !IsWormhole() && port.CanRecharge(Port::RechargeType::Fuel) && CanLand(ship);
+	return !IsWormhole() && port.CanRecharge(Port::RechargeType::Fuel, ship.IsYours()) && CanLand(ship);
+}
+
+
+
+bool Planet::CanBribe() const
```

Comment:
It feels a bit strange to have a CanBribe() function that only checks if the player can land.
`AllowLand` might be a better name for this function.

Reply:
Well, we can keep this for a follow-up PR.